recipe/meta.yaml

```
@@ -21,17 +20,18 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
   host:
     - python
     - cython
-    - numpy 2
+    - numpy >=1.25,<3
     - geos {{ geos }}
     - pip
     - setuptools >=61.0.0
     - wheel
```

https://github.com/shapely/shapely/blob/2.1.1/versioneer.py#L37
```suggestion
    - wheel
    - tomli
```